java-it-tests/common/src/main/java/queries/QueryErrorPrimingTest.java

```diff
@@ -84,4 +82,114 @@ public void testPrimingUnavailable() {
         assertEquals(4, ((CassandraResult.UnavailableStatus) status).getRequiredAcknowledgements());
         assertEquals(3, ((CassandraResult.UnavailableStatus) status).getAlive());
     }
+
+    @Test
```

Comment:
Couldn't find a clever way to share these tests between tests like I did with ErrorHandlingBehaviors using scalatest ((yay scalatest and traits!), but will see if I can find a way that doesn't involve sharing a parent class.